libs/filamat/include/filamat/MaterialBuilder.h

```diff
@@ -38,6 +38,7 @@
 #include <string>
 #include <utility>
 #include <vector>
+#include <memory>
```

Comment:
Please keep the includes in alphabetical order, thanks!

Reply:
Oh... I don't know about this. I made a #6507 for this.